55080 Order Processing - additional 'Hold' validations

tyndmar opened the pull request:
Files changed:
browsers/brwTag.w - expanded description field
oe/GetStatusDesc.p - changed "Credit Hold" to "Hold"
oe/v-ord.w:
- replaced spare-char-2 display with fillin
- added Validate Button with trigger to run pValidate
- used standard call to get status descriptions
- added pValidate procedure to run tests in oeValidate
sys/ref/dlgTagVwr.w - changed formatting
system/oeValidate.p - added test for insufficient inventoty
viewers/vwrTag.w - changed formatting

Commit:
55080 Order Processing - additional 'Hold' validations
Files changed:
browsers/brwTag.w - expanded description field
oe/GetStatusDesc.p - changed "Credit Hold" to "Hold"
oe/v-ord.w:
- replaced spare-char-2 display with fillin
- added Validate Button with trigger to run pValidate
- used standard call to get status descriptions
- added pValidate procedure to run tests in oeValidate
sys/ref/dlgTagVwr.w - changed formatting
system/oeValidate.p - added test for insufficient inventoty
viewers/vwrTag.w - changed formatting

brad-advantzware reviewed (changes requested):
See comments.  Just the logic change in oeValidate for shipto.loc

Commit:
55080 Order Processing - additional 'Hold' validations
File changed system/oeValidate.p
changed lookup of ship from warehouse to use oe-ordl.ship-id (shipto) to locate shipto record to locate ship-from warehouse.
